src/test/scala/bigquery4s/UsageExamplesSpec.scala

```diff
@@ -12,7 +12,7 @@ class UsageExamplesSpec extends FunSpec with Matchers {
 
   describe("Simple query example with publicdata") {
     it("runs") {
-      val bq = BigQuery()
+      val bq = BigQuery.fromClientSecretJson()
```

Comment:
I prefer keeping #apply method as one that calls #fromClientSecretJson internally because it looks a bit handy for me.